inc/switchtec/gas.h

```diff
@@ -67,8 +67,8 @@
 void memcpy_to_gas(struct switchtec_dev *dev, void __gas *dest,
 		   const void *src, size_t n);
 
-void memcpy_from_gas(struct switchtec_dev *dev, void *dest,
-		     const void __gas *src, size_t n);
+int memcpy_from_gas(struct switchtec_dev *dev, void *dest,
```

Comment:
Should we not have a warn_unused_result for this? Seems like it would be a good idea seeing people might skip it given that it is a memcpy like interface?

Reply:
Updated above.